packages/core/src/components/TextField.tsx

```diff
@@ -97,9 +96,6 @@ class TextField extends React.Component<Props, State> {
   componentDidMount() {
     if (this.props.defaultValue) {
       this._handleChangeText(this.props.defaultValue);
```

Comment:
I didn't see functionality to react to changes to `defaultValue` here -- are we doing that? Looks like this is just firing on mount?

Reply:
If we're going to update the react-query centric logic to use `onChange` and `value`, I don't think there's a need to react to changes in `defaultValue` (which would mirror the behavior of the react native TextInput component)

Reply:
We could add it though, I suppose? I don't think it would hurt anything